src/wizard/javascript/replay-onboarding/javascript/2.configure.md

```diff
@@ -7,22 +7,24 @@ type: language
 
 #### Configure
 
-Add the following to your SDK config. There are several privacy and sampling options available, all of which can be set via the `integrations` constructor. Learn more about configuring Session Replay in our Github [Readme](https://github.com/getsentry/sentry-replay/blob/main/README.md).
+Add the following to your SDK config. There are several privacy and sampling options available, all of which can be set via the `integrations` constructor. Learn more about configuring Session Replay by reading the [configuration docs](https://docs.sentry.io/platforms/javascript/session-replay/).
```

Comment:
On this and all these wizard files, the word "via" should be swapped for "using" since we avoid Latin words in the docs.